Assets/MixedRealityToolkit/StandardAssets/FontsSDFTextures/segoeui SDF.asset

```diff
@@ -15,7 +15,1820 @@ MonoBehaviour:
   hashCode: -1937467786
   material: {fileID: 21340371490990018}
   materialHashCode: 1402074941
-  fontAssetType: 1
+  m_Version: 1.1.0
```

Comment:
These asset files seem to have a bunch of changes that don't seem to be that related to these asmdef renames - might be worth seeing if they're needed, and if so, putting them in separately to keep each PR scoped.

Reply:
fixed by restoring from the dev branch